econpizza/__init__.py

```diff
@@ -24,6 +25,20 @@
 # create local alias
 copy = deepcopy
 
+if config.enable_jax_persistent_cache == True:
+    jax.config.update("jax_compilation_cache_dir", JAX_CACHE_DIR)
+    jax.config.update("jax_persistent_cache_min_entry_size_bytes", -1)
+    jax.config.update("jax_persistent_cache_min_compile_time_secs", 0)
+
+if config.enable_persistent_cache == True:
```

Comment:
Would it not make sence that `enable_persistent_cache` automagically enables jax cache?

Reply:
Yep, I'll make it like this.